concepts/tuples/about.md

```diff
@@ -4,37 +4,33 @@ A tuple is a versatile and efficient compound data type in Cairo, allowing you t
 
 ## Creating Tuples
 
-To create a tuple, you write a comma-separated list of values inside parentheses. Each position in the tuple has a specific type, and these types can differ from one another. Here’s an example with optional type annotations:
+To create a tuple, you write a comma-separated list of values inside parentheses. Each position in the tuple has a specific type, and these types can differ from one another.
 
 ```rust
 fn main() {
     let tup: (u32, u64, bool) = (10, 20, true);
 }
 ```
 
-In this example, the variable `tup` binds to the entire tuple, treating it as a single compound element.
```

Comment:
Don't forget to address [this change request](https://github.com/exercism/cairo/pull/106#discussion_r1700270313):
> Instead, state that a tuple can hold any other data type, even other tuples, and give a brief example.